instrumentation/elastic-opentelemetry-instrumentation-openai/src/opentelemetry/instrumentation/openai/helpers.py

```diff
@@ -112,17 +112,17 @@ def _get_span_attributes_from_wrapper(instance, kwargs):
                 elif scheme == "https":
                     span_attributes[SERVER_PORT] = 443
 
-    if frequency_penalty := kwargs.get("frequency_penalty") is not None:
+    if (frequency_penalty := kwargs.get("frequency_penalty")) is not None:
         span_attributes[GEN_AI_REQUEST_FREQUENCY_PENALTY] = frequency_penalty
-    if max_tokens := kwargs.get("max_tokens") is not None:
+    if (max_tokens := kwargs.get("max_completion_tokens", kwargs.get("max_tokens"))) is not None:
```

Comment:
TIL about deprecation of `max_tokens`:


https://platform.openai.com/docs/api-reference/chat/create

```
max_tokens
Deprecated

integer or null
Optional

The maximum number of [tokens](https://platform.openai.com/tokenizer) that can be generated in the chat completion. This value can be used to control [costs](https://openai.com/api/pricing/) for text generated via API.

This value is now deprecated in favor of max_completion_tokens, and is not compatible with [o1 series models](https://platform.openai.com/docs/guides/reasoning).
```